apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/pensjonforvalter/PensjonforvalterConsumer.java

```diff
@@ -59,100 +60,111 @@ public PensjonforvalterConsumer(
                 .build();
     }
 
-    @Timed(name = "providers", tags = {"operation", "pen_getMiljoer"})
+    @Timed(name = "providers", tags = { "operation", "pen_getMiljoer" })
     public Mono<Set<String>> getMiljoer() {
 
         return tokenService.exchange(serverProperties)
                 .flatMap(token -> new HentMiljoerCommand(webClient, token.getTokenValue()).call());
     }
 
-    @Timed(name = "providers", tags = {"operation", "popp_lagreInntekt"})
+    @Timed(name = "providers", tags = { "operation", "popp_lagreInntekt" })
     public Flux<PensjonforvalterResponse> lagreInntekter(PensjonPoppInntektRequest pensjonPoppInntektRequest) {
 
-        log.info("Popp lagre inntekt {}", pensjonPoppInntektRequest);
+        String callId = generateCallId();
```

Comment:
Nå når callId også skal logges kunne det være et alternativ å flytte loggingen inn i command-fil, istedet for å introdusere et parameter til. Det vil være enklere å lese og vedlikeholde, men dette er også OK.
(så lite kode som mulig bør være målet 😄 )